src/components/AccountsManagementComponent.jsx

```diff
@@ -159,140 +190,341 @@ class AccountsManagementComponent extends Component {
+                <Box
+                  display='flex'
+                  flexDirection='row'
+                  alignItems='center'
+                  justifyContent='space-between'
+                  style={{ width: '100%' }}
+                >
+                  <div style={{ maxWidth: '40%' }}>
+                    <Box display='flex' flexDirection='row' alignItems='center'>
+                      <Avatar
+                        style={{ borderRadius: '0px', marginRight: 10 }}
+                        src={getWalletLogo(account.walletType)}
+                      ></Avatar>
+                      <Box>
+                        <Typography variant='body2'>{account.name}</Typography>
+                        <Typography variant='caption'>
+                          {getWalletTitle(account.walletType)},{' '}
+                          {getCryptoTitle(account.platformType)}
+                        </Typography>
+                      </Box>
+                    </Box>
+                  </div>
+                  <div>
+                    <Box display='flex' flexDirection='row' justifyContent='flex-end'>
+                      <Box display='flex' flexDirection='column' alignItems='flex-end'>
+                        {account.status === accountStatus.syncing ? (
+                          <Skeleton style={{ margin: '0px', width: '100%', minWidth: '100px' }} />
+                        ) : account.assets.length === 1 ? (
+                          <Typography variant='body2'>
+                            {`${getCryptoSymbol(
+                              account.assets[0].cryptoType
+                            )} ${this.toCurrencyLocaleString(
+                              account.assets[0].balanceInStandardUnit
+                            )}`}
+                          </Typography>
+                        ) : (
+                          <Typography variant='body2'>Multiple Coins</Typography>
+                        )}
+                        <Typography variant='caption'>
+                          $ {this.toCurrencyLocaleString(account.totalMarketValue)}
+                        </Typography>
+                      </Box>
+                      <Box ml={1}>
+                        {expanded ? (
+                          <ExpandLessIcon className={classes.iconBtn} />
+                        ) : (
+                          <ExpandMoreIcon className={classes.iconBtn} />
+                        )}
+                      </Box>
+                    </Box>
+                  </div>
+                </Box>
+              </ListItem>
+              {expanded && (
+                <Box
+                  display='flex'
+                  flexDirection='column'
+                  alignItems='stretch'
+                  justifyContent='flex-start'
+                  pl={4}
+                  pr={3}
+                >
+                  <ListItem style={{ padding: 0 }}>
+                    {this.rednerAccountActionButtons(account)}
```

Comment:
on mobile, action btns do not seem to be centered

Reply:
It seems to be intended to align to the left instead of the center. 
Will align to the center now.